src/main/java/dinkplugin/SettingsManager.java

```diff
@@ -89,14 +94,39 @@ void onVarbitChanged(VarbitChanged event) {
         int value = event.getValue();
 
         if (event.getVarbitId() == CombatTaskNotifier.COMBAT_TASK_REPEAT_POPUP && isRepeatPopupInvalid(value) && config.notifyCombatTask()) {
-            warnForGameSetting(CombatTaskNotifier.REPEAT_WARNING);
+            queueVarbitCheck();
         }
 
         if (event.getVarbitId() == Varbits.COLLECTION_LOG_NOTIFICATION && isCollectionLogInvalid(value) && config.notifyCollectionLog()) {
-            warnForGameSetting(CollectionNotifier.ADDITION_WARNING);
+            queueVarbitCheck();
         }
     }
 
+    private void queueVarbitCheck() {
+        if (varbitCheckQueued.getAndSet(true))
+            return;
+
+        clientThread.invokeLater(() -> {
+            if (client.getGameState() != GameState.LOGGED_IN)
+                return false;
+
+            if (queuedTicks.getAndIncrement() < 2)
```

Comment:
```suggestion
            if (queuedTicks.getAndIncrement() < 4)
```
cba to build, being lazy with jars rn